packages/dialtone-vue2/recipes/conversation_view/wysiwyg_editor/wysiwyg_editor.vue

```diff
@@ -0,0 +1,574 @@
+     * - `end`    Sets the focus to the end of the input
+     * - `all`    Selects the whole contents of the input
+     * - `Number` Sets the focus to a specific position in the input
+     * - `true`   Defaults to `start`
+     * - `false`  Disables autofocus
+     * @values true, false, start, end, all, number
+     */
+    autoFocus: {
+      type: [Boolean, String, Number],
+      default: false,
+      validator (autoFocus) {
+        if (typeof autoFocus === 'string') {
+          return RICH_TEXT_EDITOR_AUTOFOCUS_TYPES.includes(autoFocus);
+        }
+        return true;
+      },
+    },
+
+    /**
+     * Placeholder text
+     */
+    placeholder: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Content area needs to dynamically adjust height based on the conversation area height.
+     * can be vh|px|rem|em|%
+     */
+    maxHeight: {
+      type: String,
+      default: 'unset',
+    },
+
+    /**
+     * Confirm set link button defaults.
+     */
+    confirmSetLinkButton: {
+      type: Object,
+      default: () => ({ label: 'Confirm', ariaLabel: 'Confirm set link' }),
+    },
+
+    /**
+     * Remove link button defaults.
+     */
+    removeLinkButton: {
+      type: Object,
+      default: () => ({ label: 'Remove', ariaLabel: 'Remove link' }),
+    },
+
+    /**
+     * Cancel set link button defaults.
+     */
+    cancelSetLinkButton: {
+      type: Object,
+      default: () => ({ label: 'Cancel', ariaLabel: 'Cancel set link' }),
+    },
+
+    /**
+     * Placeholder text for the set link input field
+     */
+    setLinkPlaceholder: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Show button to render text as bold
+     */
+    showBoldButton: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Show button to render text in italics
+     */
+    showItalicsButton: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Show button to underline text
+     */
+    showUnderlineButton: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Show button to strike text
+     */
+    showStrikeButton: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Show button to render list items
+     */
+    showListItemsButton: {
+      type: Boolean,
+      default: true,
+    },
+
+    /**
+     * Show add link default config.
+     */
+    showAddLink: {
+      type: Object,
+      default: () => ({
+        showAddLinkButton: true,
+        setLinkModalTitle: 'Add a link',
+        setLinkInputAriaLabel: 'Input field to add link',
+      }),
+    },
+  },
+
+  emits: [
+    /**
+     * Native focus event
+     * @event input
+     * @type {String|JSON}
+     */
+    'focus',
+
+    /**
+     * Native blur event
+     * @event input
+     * @type {String|JSON}
+     */
+    'blur',
+
+    /**
+     * Native input event
+     * @event input
+     * @type {String|JSON}
+     */
+    'input',
```

Comment:
focus and blur are outputting correctly in storybook, but seems like input is not.